hs_composite_resource/tests/data/ODM2_One_Site_One_Series_Test.csv

```diff
@@ -0,0 +1,21 @@
+ValueDateTime,Temp_DegC_Mendon
+2008-01-01 00:00:00,0.1766667
+2008-01-01 00:30:00,0.145
+2008-01-01 01:00:00,0.05666667
+2008-01-01 01:30:00,0.01666667
+2008-01-01 02:00:00,-0.01166667
+2008-01-01 02:30:00,-0.065
+2008-01-01 03:00:00,-0.1033333
+2008-01-01 03:30:00,-0.1583333
+2008-01-01 04:00:00,-0.1733333
+2008-01-01 04:30:00,-0.1733333
+2008-01-01 05:00:00,-0.18
+2008-01-01 05:30:00,-0.18
+2008-01-01 06:00:00,-0.175
+2008-01-01 06:30:00,-0.175
+2008-01-01 07:00:00,-0.18
+2008-01-01 07:30:00,-0.1716667
+2008-01-01 08:00:00,-0.17
+2008-01-01 08:30:00,-0.17
+2008-01-01 09:00:00,-0.17
+2008-01-01 09:30:00,-0.17
```

Comment:
I actually prefer more functional test approaches in nose/pytest. Mocking can get too deep and typically such overly mocking unit tests end up requiring devs to write everything twice, once for test setup/teardown and once for actual functionality. The drawback is fragility, but a lesser evil in the aggregate in my experience.